test/integration/userStatus.test.js

```diff
@@ -404,7 +404,52 @@ describe("UserStatus", function () {
           }
           expect(res).to.have.status(400);
           expect(res.body.error).to.equal(`Bad Request`);
-          expect(res.body.message).to.equal(`"currentStatus.message" is not allowed to be empty`);
+          expect(res.body.message).to.equal(
+            `The value for the 'message' field is mandatory when State is OOO for more than three days.`
+          );
+          return done();
+        });
+    });
+
+    it("Should return error when trying to update status for a past date", function (done) {
+      // marking ACTIVE from last 4 days
+      const fromDate = Date.now() - 4 * 24 * 60 * 60 * 1000;
+      chai
+        .request(app)
+        .patch(`/users/status/self`)
+        .set("cookie", `${cookieName}=${testUserJwt}`)
+        .send(generateUserStatusData("OOO", Date.now(), fromDate, "", ""))
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(400);
+          expect(res.body.error).to.equal(`Bad Request`);
+          expect(res.body.message).to.equal(
+            `The value for the 'from' field must be a date that is after the today's date.`
```

Comment:
can we keep it similar to line 387 or change line 387 to maintain consistency

Reply:
changed line 387 to maintain consistency.